www/js/survey/enketo/enketo-add-note-button.js

```diff
@@ -42,23 +30,37 @@ angular.module('emission.survey.enketo.add-note-button',
 
   const updateLabel = () => {
     const localeCode = $translate.use();
-    if ($scope.notesConfig?.['filled-in-label'] && trip.tripAddition?.length > 0) {
+    if ($scope.notesConfig?.['filled-in-label'] && timelineEntry.additions?.length > 0) {
```

Comment:
Question for @shankari, should it be standard for every program that if they are using notes-addition buttons, they MUST require a `'not-filled-in-label'` and they only need to require a `'filled-in-label'` if they want to?

Is it safe to assume this as the determining factor for whether we will display `'filled-in-label'`?

Reply:
We don't have a use case for `filled-in-label` right now, so we haven't actually implemented the logic for the label to change.

I'm not sure what the use case would be exactly - filling in the label makes sense for 1:1 user inputs, but not for 1:n additions because I imagine the label will always be something like "Add X".

After one X is added, would there ever be a situation where we want it to change to say "Add More X"?